src/common/PDFOptions.ts

```diff
@@ -24,6 +24,9 @@ export interface PDFMargin {
   right?: string | number;
 }
 
+/**
```

Comment:
I wonder what is our stance of making changes that doesn't directly relate to the aim of the PR.

For example, why this change to the documentation introduced I'm not able to understand from the title.

I'd prefer to keep unrelated changes in separate PRs, commits. WDYT?

Reply:
This is related. The `@microsoft/api-extractor` warns about this interface being exported without a comment above.

Reply:
Oh, then it would be great to have this information provided in the PR description :)